web/sites/default/config/field.field.taxonomy_term.council_committee.field_committee_chairs.yml

```diff
@@ -0,0 +1,34 @@
+uuid: c99732c1-d81c-43bb-8e57-364afde8eba2
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.taxonomy_term.field_committee_chairs
+    - group.type.elected_official
+    - taxonomy.vocabulary.council_committee
+  module:
+    - tmgmt_content
+third_party_settings:
+  tmgmt_content:
+    excluded: false
+id: taxonomy_term.council_committee.field_committee_chairs
+field_name: field_committee_chairs
```

Comment:
If possible could we rename this field machine name to `field_committee_members`? It might require a bit of work of deleting this field and creating a new one, but it would be good to nail down the machine name before it's merged and we start entering data.